### ISSUE TITLE
Base Metals conflict

### ISSUE THREAD

**Issue by vivredraco** [PersonTheCat/ore_stone_variants#66]
I seem to be running into a conflict between Base Metals and Ore Stone Variants:

The game crashed whilst initializing game
Error: java.lang.NullPointerException: Error: assets/basemetals/textures/blocks/zinc_ore.png could not be found.

I'm using Minecraft 1.12.2, BaseMetals-1.12-2.5.0-beta7.311.jar, and ore_stone_variants-4.4.jar, through the Twitch launcher. No other mods are installed (except OreSpawn-1.12-3.3.1.170.jar, which is a Base Metals dependency). This error does not occur when Base Metals is installed by itself, only when Ore Stone Variants is also present.

I checked the Base Metals JAR, and confirmed that file path actually does exist in there.

**Comment by vivredraco** [comment]
As discussed on CurseForge this is likely a version conflict. I did try a couple different versions of each plugin, but I wasn't able to find versions that worked together.

Let me know if there's anything I could look at that might help resolve this issue. Thanks.

**Comment by PersonTheCat** [comment]
Thanks for posting this issue here. I appreciate that. I tried to replicate this issue using MC 1.12.2, BaseMetals 1.12-2.5.0 beta 7.311, OSV 4.4, and Forge 1.12.2 14.23.5.2824 (both with and without OreSpawn), but did not experience a crash. I made sure to delete resources.zip to verify that OSV indeed was able to retrieve the zinc ore texture in order to generate an overlay. 
So, I'm not sure what's going on. :/ Which Forge version are you using?

**Comment by vivredraco** [comment]
Looks like I'm on forge-14.23.5.282**3**. Confirmed working in 2824.

Reviewing the Forge site, I see two versions listed for download: 2768 (recommended) and 2836 (latest). Twitch appears to default to latest. New builds must come out fast, as I only created this profile like a month ago.

As long as I'm at it and in the interest of thoroughness, I went ahead and tested with both versions Forge currently lists for download. Both work.

Just out of curiosity, I tried 2822. It also worked. So it looks like I just happened to be on literally the only version of Forge this didn't work with.

Thanks for your assistance.
